server/controllers/refer/courseParticipationsController.ts

```diff
@@ -249,10 +251,10 @@ export default class CourseParticipationsController {
     }
   }
 
-  private async courseParticipationsWithNames(
+  private async presentCourseParticipations(
```

Comment:
This is also in `referralController`.  Are we happy to have this in a util file and use the services in there?

Reply:
Ahh, the original method will be removed in the branch I'm working on, which we'd hoped would make this problem go away (hence the temporary duplication). The reason the original method was in the controllers and not utils is because it's calling services - it felt a bit weird to be doing that kind of async work in utils. But it looks like this introduces a new service call, so we'll have to work out some way to DRY it up?

Reply:
Yeah that's my feeling too. I think like we've mentioned [here](https://github.com/ministryofjustice/hmpps-accredited-programmes-ui/pull/279#discussion_r1362364359) might be the way to go.